features/ProductNews.feature

```diff
@@ -0,0 +1,54 @@
+@owner:derek
+@author:lilla
+
+Feature: Product News Page
+
+    Display the Product News Page with cards to navigate to news.
+
+    @motivating
+    Scenario: Product news updates can be filtered by type
+        Given a user selects from the type dropdown
+        Then they can see the following items
+
+            | postType         |
+            | All Posts        |
+            | Feature Releases |
+            | Monthly Updates  |
+            | Product Updates  |
+
+    @motivating
+    Scenario: Product news updates can be filtered by date
+        Given a user selects from the date dropdown
+        Then they can see the following items
+
+            | date          |
+            | From All-time |
+            | Last 30 days  |
+            | Past Year     |
+
+    @motivating
+    Scenario: News cards are showing on the page in one column regardless of the screen size
```

Comment:
When your scenario title mentions that something is displayed regardless of screen sizes, that detail should be included within your spec as well. (scenario outline)